lib/broccoli/ember-app.js

```diff
@@ -1341,15 +1321,17 @@ EmberApp.prototype.styles = function() {
     }.bind(this));
   }
 
-  this.import('vendor/addons.css');
-
   var vendorStyles = [];
   for (var outputFile in this._styleOutputFiles) {
+    var isMainVendorFile = outputFile === this.options.outputPaths.vendor.css;
     var headerFiles = this._styleOutputFiles[outputFile];
+    var inputFiles = isMainVendorFile ? ['modules/**/*.css'] : [];
```

Comment:
bower_components is here as bower_components/ and vendor is at vendor/. This concat does not include either of them (which is why we use `modules/**`), but makes them available for you if you had app.import'ed from them (which is how app.import had always worked).